src/Reflow/Parser.hs

```diff
@@ -22,12 +22,20 @@ parseFile t = either (return . parserError t) id
 parseContent :: Parser [Content]
 parseContent = do
     h <- option [] (try $ many header)
-    c <- many (quoted <|> try codeBlock <|> try pgpBlock <|> normal)
+    c <- many (blank <|> quoted <|> try codeBlock <|> try pgpBlock <|> normal)
     void eof
     return $ h <> c
 
 normal :: Parser Content
-normal = Normal <$> singleLine
+normal = do
+    ws <- many (char ' ')
+    l <- singleLine
+    return $ Normal (length ws) l
+
+blank :: Parser Content
+blank = do
+    _ <- try $ trailingWhitespace >> eol
+    return Blank
```

Comment:
FWIW: there's a way to do this "do a thing, discard that and return  this other thing" pattern, I think it's like `try trailingWhitespace >> eol *> Blank` or something.